signimage/go.mod

```diff
@@ -0,0 +1,9 @@
+module readkmod
```

Comment:
Any special reason this is not called `signimage`?

Reply:
no just a left over from an older name, but this go.mod hass been removed so fixed by default :)